controllers/nc_news.controller.js

```diff
@@ -1,8 +1,17 @@
-const { selectTopics } = require("../model/nc_news.models");
-const endpoints = require("../endpoints.json")
+const { selectTopics, selectArticleById } = require("../model/nc_news.models");
+const endpoints = require("../endpoints.json");
 
 function getAPI(request, response) {
-    return response.status(200).send(endpoints)
+	return response.status(200).send(endpoints);
+}
+
+function getArticleById(request, response, next) {
+	const articleId = Number(request.params.article_id);
+	return selectArticleById(articleId)
```

Comment:
You could use destructuring to extract the article id from req.params like this: 
 `const { article_id } = req.params`